homeassistant/components/sensor/arlo.py

```diff
@@ -133,6 +164,24 @@ def update(self):
             except TypeError:
                 self._state = None
 
+        elif self._sensor_type == 'temperature':
+            try:
+                self._state = self._data.ambient_temperature
+            except (AttributeError, TypeError):
```

Comment:
I had these in there because the library was in some cases bubbling up an `AttributeError` but in further testing, it appears to be the result of an issue that was fixed in `pyarlo==0.1.9` where the method to get the ambient sensor data was not being called on update. I've removed them and will push.